dev-python/py-smbpasswd/py-smbpasswd-1.0.2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Module capable of generating both LANMAN and NT password hashes, for e.g. Samba"
+HOMEPAGE="https://barryp.org/software/py-smbpasswd/"
+SRC_URI="https://barryp.org/static/media/software/download/${PN}/${PV}/${PN#py-}-${PV}.tgz"
```

Comment:
You're using `${PN#py-}-${PV}` and `${P#py-}` interchangeably. Stick to one, and put it as MY_P ;-).